src/audio/commands/Music/exportqueue.ts

```diff
@@ -1,30 +1,32 @@
-import { MusicCommand, QueueEntry, requireQueueNotEmpty } from '#lib/audio';
+import { AudioCommand, QueueEntry, RequireQueueNotEmpty } from '#lib/audio';
 import { LanguageKeys } from '#lib/i18n/languageKeys';
 import type { GuildMessage } from '#lib/types/Discord';
 import { map, prependIfNotNull, take } from '#utils/common';
+import { getAudio } from '#utils/functions';
 import { ApplyOptions } from '@sapphire/decorators';
+import { send } from '@skyra/editable-commands';
 import { serialize } from 'binarytf';
 
 export const maximumExportQueueSize = 100;
 
-@ApplyOptions<MusicCommand.Options>({
+@ApplyOptions<AudioCommand.Options>({
 	aliases: ['eq'],
-	cooldown: 10,
+	cooldownLimit: 10,
 	description: LanguageKeys.Commands.Music.ExportQueueDescription,
 	extendedHelp: LanguageKeys.Commands.Music.ExportQueueExtended,
-	permissions: ['ATTACH_FILES'],
-	runIn: ['text', 'news']
+	requiredClientPermissions: ['ATTACH_FILES'],
+	runIn: ['GUILD_ANY']
```

Comment:
Enumssss :(

Reply:
Mate that's a property on framework CommandOptions. Should've made the review comment back then smh.